rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
- [ ] You are required to create `getters & setters` for the `person & book` so using `attr_accessor` would be appropriate in this instance, Please kindly add this to your code and resubmit for approval :+1: :sparkles: